source/commands.txt

```diff
@@ -1,7 +1,7 @@
 .. _vsce-commands:
 
 =====================
-|vsce-short| Commands
+|vscode-short| Commands
 =====================
```

Comment:
[nit] since we changed the variable here, we want to make sure the header syntax has enough `=`s to cover the text. a new build error is popping up bc of this!